### PR TITLE
Add `helm` CLI to image

### PR DESCRIPTION
**Pull request by dhaiducek**
ref: https://issues.redhat.com/browse/ACM-9060

* [x] I have taken backward compatibility into consideration.

### CODE REVIEW

**Comment by xiangjingli** [comment]
@dhaiducek  Just unhold it once your team makes the decision.  We are no problem to bundle the helm CLI in the subscription image

File: build/Dockerfile [hunk 1]
```diff
@@ -29,14 +32,14 @@ ENV OPERATOR=/usr/local/bin/multicluster-operators-subscription \
     POLICY_GEN_ENABLE_HELM=true
 
 # install operator binary
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/multicluster-operators-subscription ${OPERATOR}
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/multicluster-operators-placementrule /usr/local/bin
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/uninstall-crd /usr/local/bin
-COPY --from=plugin-builder go/src/github.com/open-cluster-management/multicloud-operators-subscription/build/_output/bin/appsubsummary /usr/local/bin
```
Comment (xiangjingli) [inline build/Dockerfile:35]: The appsubsummary cli is required.

Comment (dhaiducek) [reply build/Dockerfile:35]: Oops! Didn't mean to take that one out! Fixed!

**Review by mprahl — APPROVED** [review]

**Pull request merged**